### PR TITLE
Edit FindCommand implementation DG

### PR DESCRIPTION
**Pull request by BlazeChron**
Added uml diagrams

### CODE REVIEW

**Review by jinhanjinhan — COMMENTED** [review]
Overall looks good

File: docs/diagrams/FindSequenceDiagram-Logic.puml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+activate AddressBookParser
+
+create FindCommandParser
+AddressBookParser -> FindCommandParser
+activate FindCommandParser
+
+FindCommandParser --> AddressBookParser
+deactivate FindCommandParser
+
+AddressBookParser -> FindCommandParser : parse(find)
```
[the remark this answers is not in the document]
Comment (BlazeChron) [reply docs/diagrams/FindSequenceDiagram-Logic.puml:28]: Would something like this be ok? It feels a bit awkward to represent certain objects in text like argMultiMap
image

image

[the remark this answers is not in the document]
Comment (jinhanjinhan) [reply docs/diagrams/FindSequenceDiagram-Logic.puml]: yep i think this is great

**Review by jinhanjinhan — APPROVED** [review]
Looks good

**Pull request merged**